akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala

```diff
@@ -41,11 +39,12 @@ final class Http2Ext(private val config: Config)(implicit val system: ActorSyste
    * Handle requests using HTTP/2 immediately, without any TLS or negotiation layer.
    */
   private def bindAndHandleWithoutNegotiation(
-    handler:   HttpRequest ⇒ Future[HttpResponse],
-    interface: String, port: Int = DefaultPortForProtocol,
-    settings:    ServerSettings = ServerSettings(system),
-    parallelism: Int            = 1,
-    log:         LoggingAdapter = system.log)(implicit fm: Materializer): Future[ServerBinding] = {
+    handler:     HttpRequest ⇒ Future[HttpResponse],
+    interface:   String,
+    port:        Int,
+    settings:    ServerSettings,
+    parallelism: Int,
+    log:         LoggingAdapter)(implicit fm: Materializer): Future[ServerBinding] = {
```

Comment:
Default value of log was removed, was it never used?

Reply:
Not for this private method